src/test/system_tests.cpp

```diff
@@ -28,22 +28,20 @@ BOOST_AUTO_TEST_CASE(run_command)
 #ifdef WIN32
         const UniValue result = RunCommandParseJSON("cmd.exe /c echo {\"success\": true}");
 #else
-        const UniValue result = RunCommandParseJSON("echo {\"success\": true}");
+        const UniValue result = RunCommandParseJSON(R"(echo '{"success": true}')");
 #endif
         BOOST_CHECK(result.isObject());
         const UniValue& success = result.find_value("success");
         BOOST_CHECK(!success.isNull());
         BOOST_CHECK_EQUAL(success.get_bool(), true);
     }
+#ifdef WIN32
```

Comment:
Why is this test being scoped to only Windows?

Reply:
On other platforms, `execl()` might fail only if there is problem with `/bin/sh`. I can't see any trivial way to adjust the test to trigger a failure. Any suggestions?